src/redux/reducers/sliderReducer.js

```diff
@@ -0,0 +1,56 @@
+import {
+  GET_HIGH_STOCK_CONFIG_REQUEST, GET_HIGH_STOCK_CONFIG_REQUEST_ERROR,
+  GET_HIGH_STOCK_CONFIG_REQUEST_SUCCESS, SET_HIGH_STOCK_CONFIG_REQUEST
+} from '../../common/constants/highStockConstants';
+
+const initialState = {
+  isLoading: false,
+  initialConfig: {
+    min: null,
+    max: null,
+    defaultStart: null,
+    defaultFinish: null
+  },
+  error: null,
+};
+
+export const sliderReducer = (state = initialState, action) => {
+  switch (action.type) {
+
+    case GET_HIGH_STOCK_CONFIG_REQUEST:
+      return {
+        ...state,
+        isLoading: true
+      };
+
+    case SET_HIGH_STOCK_CONFIG_REQUEST:
+      return {
+        ...state,
+        initialConfig: {
+          ...state.initialConfig,
+          defaultStart: action.payload?.start_date || null,
```

Comment:
what is this `action.payload?.start_date`?
use `isNil` from the `lodash` library and ternary operator